main.go

```diff
@@ -290,49 +264,122 @@ func main() {
 	var port int
 	var le string
 	var gs bool
-
+	var tlsPort int
+	var tlsCert string
+	var vers bool
 	pwd := getpwd()
 
-	flag.IntVar(&port, "p", 8000, "bind port")
-	flag.StringVar(&le, "l", "", "enable TLS with Let's Encrypt for the given domain name. Port = port + 1 ")
-	flag.BoolVar(&gs, "g", false, "generate and use a self signed certificate")
-	flag.Parse()
-
-	if le != "" {
-		var srv http.Server
-		http2.ConfigureServer(&srv, new(http2.Server))
+	flag.Usage = func() {
+		w := os.Stderr
+		for _, arg := range os.Args {
+			if arg == "-?" || arg == "-h" {
+				w = os.Stdout
+				break
+			}
+		}
+		fmt.Fprintf(w, "simple-httpd version: %s\n", name+pathSeperator+version)
```

Comment:
Would there be any benefit of putting this formatted block of code in a `const` and avoid all of the writes?